keras/backend/jax/core.py

```diff
@@ -314,3 +315,18 @@ def unstack(x, num=None, axis=0):
         jax.lax.index_in_dim(x, i, axis, keepdims=False)
         for i in range(x.shape[axis])
     ]
+
+
+@contextlib.contextmanager
+def device(device):
+    if isinstance(device, str):
```

Comment:
Also, let's validate the string format so the error message is clear if users do something not supported like "cuda"